### DISCUSSION TITLE
Why Ubuntu 22 software doesn't have Truffle Suite?

### DISCUSSION THREAD

**Discussion by AftabAhmedAbro** [ConsenSys-archive/truffle#5711] [General Q&A]
As I am wondering and facing so many issues installing Truffle suite on an Ubuntu machine isn't it easy to make a suite available on Ubuntu 22 software that doesn't have Truffle Suite?

**Comment by FlaviuWeb3** [comment]
Hi @AftabAhmedAbro could you please share with us the errors you're encountering?

**Comment by AftabAhmedAbro** [comment]
As I am trying to install the truffle extension on Visual Studio Code it is continuously running but unable to install truffle suite and ganache

**Comment by insanity54** [comment]
I don't use VS Code but I'm on Ubuntu 22.04 and I had no issues installing trufflesuite using [yarn](https://yarnpkg.com/getting-started).

`yarn add truffle`

I'm assuming that this problem has less to do with Ubuntu and more to do with how VS Code is abstracting the process of package installation. Is there a way to show output of what VS Code is doing in the background? Maybe run VS Code in a shell?

**Reply by apdan510** [reply]
thanks for this! Going to run some tests

**Comment by apdan510** [comment]
@AftabAhmedAbro I was able to install and run a few tests on Ubuntu 22. Let us know if you are still having issues. 

Thanks!
Alex | Truffle Support Team

**Reply by AftabAhmedAbro** [reply]
As I was unable to install Truffle with terminal and App software where truffle was not uploaded, so I updated the npm on my system and then it gets resolved
